Pod/Classes/Optional Data Sources/VOKDefaultPagingAccessory.m

```diff
@@ -40,9 +38,77 @@ - (void)didMoveToSuperview
+                                                                  attribute:NSLayoutAttributeCenterY
+                                                                  relatedBy:NSLayoutRelationEqual
+                                                                     toItem:containerView
+                                                                  attribute:NSLayoutAttributeCenterY
+                                                                 multiplier:1
+                                                                   constant:0]];
+        [containerView addConstraint:[NSLayoutConstraint constraintWithItem:self.indicator
+                                                                  attribute:NSLayoutAttributeCenterY
+                                                                  relatedBy:NSLayoutRelationEqual
+                                                                     toItem:containerView
+                                                                  attribute:NSLayoutAttributeCenterY
+                                                                 multiplier:1
+                                                                   constant:0]];
+
+        // Make the container fill this view, and center it horizontally
+        [self addConstraint:[NSLayoutConstraint constraintWithItem:containerView
+                                                         attribute:NSLayoutAttributeTop
+                                                         relatedBy:NSLayoutRelationEqual
+                                                            toItem:self
+                                                         attribute:NSLayoutAttributeTop
+                                                        multiplier:1
+                                                          constant:0]];
+        [self addConstraint:[NSLayoutConstraint constraintWithItem:containerView
+                                                         attribute:NSLayoutAttributeBottom
+                                                         relatedBy:NSLayoutRelationEqual
+                                                            toItem:self
+                                                         attribute:NSLayoutAttributeBottom
+                                                        multiplier:1
+                                                          constant:0]];
+        [self addConstraint:[NSLayoutConstraint constraintWithItem:containerView
+                                                         attribute:NSLayoutAttributeCenterX
+                                                         relatedBy:NSLayoutRelationEqual
+                                                            toItem:self
+                                                         attribute:NSLayoutAttributeCenterX
+                                                        multiplier:1
+                                                          constant:0]];
```

Comment:
Consider constraining `containerView`'s leading/trailing to be greater/less than `self`'s leading/trailing (to keep the container and thus the spinner and label from extending outside `self`.